data/json/items/tool/misc.json

```diff
@@ -345,6 +434,22 @@
     "weight": "600 g",
     "flags": [ "TRADER_AVOID" ]
   },
+  {
+    "type": "TOOL",
+    "id": "eac_gasket_plastic",
+    "name": "plastic gasket",
+    "name_plural": "plastic gaskets",
+    "description": " A single-use plastic gasket.",
+    "symbol": "(",
+    "color": "dark_gray",
+    "looks_like": "plastic_chunk",
+    "weight": "20 g",
+    "volume": "250 ml",
```

Comment:
That's a huge gasket, how big is this intended to be? That would be something you'd see in, like, large industrial machinery

Reply:
Good point. I'll have to weigh a few and see what they r and then calculate volume

Reply:
theres about 3 different size gaskets for the closed look extractors, 6" wide diameter, 4" wide, and 2" wide. the 6" wide one is pretty large, we could probably drop the volume down to 180ml but the weight should stay the same or increased actually. still havent weighed one yet, ill get to that later this evening.

Reply:
whats the weight/volume on a plastic chunk? i didnt think about being able to get more than 1 gasket out of a plastic chunk but that might be a possiblilty

Reply:
theres about 3 different size gaskets for the closed look extractors, 6" wide diameter, 4" wide, and 2" wide. the 6" wide one is pretty large, we could probably drop the volume down to 180ml but the weight should stay the same or increased actually. still havent weighed one yet, ill get to that later this evening.

Reply:
I don't think we need a whole ton of different gaskets. Just choose a medium number and abstract it from there, as we do with things like hinges.

Reply:
Check it now and tell me what you think. i dropped weight down to 5g and volume down to 60ml, also gave a small "//" description.

Reply:
oops i mean 30ml :-)